app/controllers/avo/actions_controller.rb

```diff
@@ -65,6 +70,8 @@ def set_query
       resource_ids = action_params[:fields][:avo_resource_ids].split(",")
 
       @query = decrypted_query || (resource_ids.any? ? @resource.find_record(resource_ids, params: params) : [])
+
+      TestBuddy.hi("Query count: #{@query.size}")
```

Comment:
Ensure that `TestBuddy` messages are triggered exclusively from the dummy app located in `spec/dummy/app`. Otherwise, this could slightly degrade performance across all apps, even though it’s solely for testing purposes on our side.

```suggestion
```